web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -223,6 +223,7 @@ export class AccountService {
         resolve(account);
       } catch (e) {
         console.log(e);
+        resolve(account);
       }
```

Comment:
This call should always resolve the account, because `getAccount` "fails" for new users.

Reply:
Exactly